docs/user/openstack/README.md

```diff
@@ -498,6 +498,14 @@ Another useful thing to check is that the ignition configurations are only avail
 curl https://<loadbalancer ip>:22623/config/master --insecure
 ```
 
+## Refreshing a CA Certificate
+
+If you ran the installer with a custom CA certificate, then this certificate can be changed while the installer is running as needed. To chage your certificate, edit the value of the `ca-cert.pem` key in the `cloud-provider-config` configmap with a valid PEM certificate.
```

Comment:
The "while the installer is running" phrase makes me think you would want to do this during the installation process.

But I think this is more for updating the cert after the installation finished, during normal operation of the OpenShift cluster? Would you mind changing it to something like "while the cluster is running"?

Sorry if I completely misunderstood what this is for!

Reply:
Ah good catch, I meant "while the cluster is runnning."